_includes/configuration/toolbar.md

```diff
@@ -17,6 +17,8 @@ tinymce.init({
 });
 ```
 
+> **Note**: The toolbar buttton groups defined in the `toolbar` setting will affect which toolbar buttons will be shown when the toolbar is wider than the editor. For more information, see: [the `toolbar_mode` setting]({{site.baseurl}}/configure/editor-appearance/#toolbar_mode).
```

Comment:
does it matter that this is specifically if toolbar_mode is floating or sliding?

Reply:
Also, there's a typo here `buttton` (one too many t's).

Reply:
May of been fixed by: https://github.com/tinymce/tinymce-docs/pull/1612/commits/85c06bb5d09d833d450f339a14642ad85c3ad1db